### PR TITLE
reorganize component to include multi-file support, include example

### PR DESCRIPTION
**Pull request by rsnancollas**
This re-organizes the way components are organized to reflect the multi-file naming scheme we came up with.

### CODE REVIEW

File: part-spec/component.json
Comment (asadate) [inline part-spec/component.json]: Typo: "CoreProperties" here and in the file

**Review by asadate — APPROVED** [review]
Some typos to fix. Aligned with what we discussed. I'll approve the request but you may want to wait for the WG members' feedback  before merging

File: part-spec/component.json
Comment (rsnancollas) [reply part-spec/component.json]: whoops. thanks

**Review by asadate — COMMENTED** [review]
All changes addressed.

**Pull request merged**
